pkg/cmd/roachprod/main.go

```diff
@@ -1089,6 +1089,15 @@ func main() {
 		fmt.Sprintf("The cloud provider(s) to use when creating new vm instances: %s", vm.AllProviderNames()))
 	createCmd.Flags().BoolVar(&createVMOpts.GeoDistributed,
 		"geo", false, "Create geo-distributed cluster")
+
+	defaultMap := make(map[string]string)
+
+	createCmd.Flags().StringToStringVar(&createVMOpts.CustomLabelMap,
+		"label", defaultMap,
+		"The label(s) to be used when creating new vm instances, must be in '--label name=value' format and value "+
+			"can't be empty string after trimming space, a value that has space must be quoted by single quotes. "+
+			"Examples: usage=cloud-report-2021, namewithspaceinvalue='s o s'")
```

Comment:
Will values with spaces actually work with the way these labels are getting passed to the cloud providers?